src/web/core/src/main/java/org/geoserver/web/ServiceDescriptionProvider.java

```diff
@@ -132,8 +132,9 @@ protected ServiceDescription description(
     }
 
     /**
-     * Gets the name of the {@code version} parameter for the service. This will usually be {@code version}, but
-     * some (i.e. WCS 2+) it will be {@code acceptversions}. To overrided by subclasses.
+     * Gets the name of the {@code version} parameter for the service. This will usually be {@code
```

Comment:
I think you should rebase to main, and this change will disappear.

Reply:
done